README.md

```diff
@@ -0,0 +1,6 @@
+Various GO Lang examples to help you get a better understanding of the langauage
+
+To test any of the examples:
+
+go run filename.go
+
```

Comment:
I think we should get rid of the comments in the code then explain what each file does in the readme and how

Reply:
Smart idea but then comments might help readers get better insights.